src/main/java/frc/robot/LogitechGamingPad.java

```diff
@@ -26,230 +26,207 @@
 	/**
 	 * The Joystick class is utilized for simplicity.
 	 */
-	private Joystick gamepad;
-
-	/**
-	 * Construct an instance of a Logitch Gaming Pad. The Logitech Gaming Pad index
+  private Joystick gamepad;
+
+	// Constants
+  private static final int LEFT_ANALOG_X_AXIS = 0;
+  private static final int LEFT_ANALOG_Y_AXIS = 1;
+  private static final int RIGHT_ANALOG_X_AXIS = 4;
+  private static final int RIGHT_ANALOG_Y_AXIS = 5;
+	
+  private static final int LEFT_BUMPER = 9;
+  private static final int RIGHT_BUMPER = 10;
+  private static final int LEFT_TRIGGER_BUTTON = 2;
+  private static final int RIGHT_TRIGGER_BUTTON = 3;
+  private static final int A_BUTTON = 1;
+  private static final int B_BUTTON = 2;
+  private static final int X_BUTTON = 3;
+  private static final int Y_BUTTON = 4;
+  private static final int BACK_BUTTON = 7;
+  private static final int START_BUTTON = 8;
+	/**
+	 * construct an instance of a Logitch Gaming Pad. The Logitech Gaming Pad index
 	 * is the usb port on the driver station.
 	 *
 	 * @param port The port on the driver station that the gamepad is plugged
 	 *        into.
 	 */
-
-	public LogitechGamingPad(int port)
-	{
-		super(0);
-		gamepad = new Joystick(port);
-	}
+	
+  public LogitechGamingPad(int usbPort) {
+    super(0);
+    gamepad = new Joystick(usbPort);
+  }
 
 	// The following methods are get methods specific to the Logitech Gaming Pad.
 	// Use these methods to get button values.
 
-	/**
-	 * Get the X value of the left analog.
-	 * 
-	 * @return The left X value.
-	 */
-	public double getLeftAnalogX()
-	{
-		return getLeftAnalogX();
-	}
-
-	/**
-	 * Get the Y value of the left analog.
-	 * 
-	 * @return The left Y value.
-	 */
-
-	public double getLeftAnalogY()
-	{
-		return getLeftAnalogY();
-	}
-
 	/**
 	 * Read the state of the left analog button.
 	 * This is the button part of the analog.
```

Comment:
might be good to change this comment to talk about the "left bumper" instead of "left analog button" since the diff terminology is a bit confusing